app/javascript/packs/stylesheets/custom_table.scss

```diff
@@ -54,3 +54,21 @@
 .table-smaller-font {
   font-size: 0.9em;
 }
+
+.patients-table .table-hover tbody tr:hover {
+  background-color: #e4e4e4 !important;
```

Comment:
think these colors are important enough to put in the `constants.scss` file?

Reply:
Not really.